qa/suites/rados/thrash/workloads/snaps-few-objects-redelete.yaml

```diff
@@ -0,0 +1,19 @@
+overrides:
```

Comment:
This failure also looks related to this test:

/a/yuriw-2024-04-11_17:03:54-rados-wip-yuri6-testing-2024-04-02-1310-distro-default-smithi/7652505
```
2024-04-12T01:51:20.020 INFO:tasks.thrashosds.thrasher:Traceback (most recent call last):
  File "/home/teuthworker/src/github.com_ceph_ceph-c_a5074d4516d566e9d8b6aec912f26afd099de101/qa/tasks/ceph_manager.py", line 190, in wrapper
    return func(self)
  File "/home/teuthworker/src/github.com_ceph_ceph-c_a5074d4516d566e9d8b6aec912f26afd099de101/qa/tasks/ceph_manager.py", line 1483, in _do_thrash
    self.choose_action()()
  File "/home/teuthworker/src/github.com_ceph_ceph-c_a5074d4516d566e9d8b6aec912f26afd099de101/qa/tasks/ceph_manager.py", line 1321, in <lambda>
    self.inject_pause(key,
  File "/home/teuthworker/src/github.com_ceph_ceph-c_a5074d4516d566e9d8b6aec912f26afd099de101/qa/tasks/ceph_manager.py", line 1065, in inject_pause
    self.ceph_manager.set_config(the_one, **{conf_key: duration})
  File "/home/teuthworker/src/github.com_ceph_ceph-c_a5074d4516d566e9d8b6aec912f26afd099de101/qa/tasks/ceph_manager.py", line 2093, in set_config
    self.wait_run_admin_socket(
  File "/home/teuthworker/src/github.com_ceph_ceph-c_a5074d4516d566e9d8b6aec912f26afd099de101/qa/tasks/ceph_manager.py", line 2050, in wait_run_admin_socket
    raise Exception('timed out waiting for admin_socket '
Exception: timed out waiting for admin_socket to appear after osd.4 restart
```